LandSandBoat/server
# [Quest] Implement In Search of the Truth

**_I affirm:_**

## What does this pull request do?

* Added In Search of the Truth
* Added item Gramary Cape
* Added Default Actions to Quest QMs
* Added Text IDs for the quest In Search of the Truth

## Steps to test these changes

-- Be past CoP 3-5
Talk to Tressia
Talk to: Raminey Zadant Fouagine Noam in any order
Talk to Tress: 1 2 4 3
Optional: Talk to Mengrenaux and Chemioue
Talk to Tressia
1 2 4 2
Check QM1 Obtain KI
Check QMs in order 1 -> 2 -> 3 -> 4 -> 5
5 triggers a CS "completes the quest"
Talk to Tressia
Talk to Ondieulix to receive reward
Talk to Tressia to test new default action

-- Tavnazian Safehold           !zone 26
-- Tressia                      !pos 87.527 -33.991 70.481
-- Raminey                      !pos 82.396 -35.972 49.707
-- Zadant                       !pos -10.532 -21.996 -15.71
-- Fouagine                     !pos 25.313 -35.911 -20.944
-- Noam                         !pos -70.17 -11.264 6.788
-- Mengrenaux                   !pos 72.844 -34.25 63.924
-- Chemioue                     !pos 82.041 -33.964 67.636
-- Ondieulix                    !pos 6.775 -24.955 65.399
-- qm1_in_search_of_truth       !pos -73.005 -11.123 11.842
-- qm2_in_search_of_truth       !pos -80.654 -19.988 65.902
-- qm3_in_search_of_truth       !pos -61.469 -22.007 44.123
-- qm4_in_search_of_truth       !pos -26.099 -21.955 26.298
-- qm5_in_search_of_truth       !pos -4.733 -22.733 23.587
-- KI Shaded Cruse              718

## Captures
Main source
https://youtu.be/ZcZqUOKzxPQ

After Quest is completed new default action of Tressia:
https://youtu.be/6F7yhKKS82U

Additional Captures:
https://youtu.be/Kj_uxt30B00
https://youtu.be/_0Fw-pK23hI
